themes/hugo-steam-theme/layouts/partials/head.html

```diff
@@ -19,6 +19,17 @@
       <link href="{{ .RSSLink }}" rel="feed" type="application/rss+xml" title="{{ .Site.Title }}" />
     {{ end }}
 
+    <meta property="og:type" content="article" />
+    <meta property="og:title" content="{{ .Title }}"/>
+    <meta property="og:description" content="{{ .Description | markdownify }}" />
+    <meta name="og:image" content="https:{{ .Site.BaseURL }}{{.Params.image}}" />
+
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:site" content="@sourcedtech" />
+    <meta name="twitter:creator" content="@sourcedtech" />
+    <meta name="twitter:title" content="{{ .Title }}"> 
+    <meta name="twitter:image" content="https:{{ .Site.BaseURL }}{{.Params.image}}" />
```

Comment:
From twitter cards docs:
>_When the Twitter card processor looks for tags on a page, it first checks for the Twitter-specific property, and if not present, falls back to the supported Open Graph property. This allows for both to be defined on the page independently, and minimizes the amount of duplicate markup required to describe content and experience._

https://developer.twitter.com/en/docs/tweets/optimize-with-cards/guides/getting-started

I think we can drop the `twitter:title` and `twitter:image`. This is not a strong opinion, but a nice to have.

Reply:
I think I'd rather keep "just in case"